src/analyzer.rs

```diff
@@ -132,9 +135,25 @@ impl<A> Default for AnalyzerConfig<'_, A> {
             .set_tokenizer(LegacyMeilisearch));
 
         // Chinese script specialized pipeline
+        
         pipeline_map.insert((Script::Mandarin, Language::Other), Pipeline::default()
             .set_pre_processor(ChineseTranslationPreProcessor)
             .set_tokenizer(Jieba::default()));
+        
+        // Japanese script specialized pipeline
+        // TODO: define dict path for japanese
+
+        let mut tokenizer = LinderaTokenizer::new(Mode::Normal, "");
+
+        pipeline_map.insert((Script::Katakana, Language::Other), Pipeline::default()
+            .set_tokenizer(Lindera { tokenizer }));
+
+        pipeline_map.insert((Script::Hiragana, Language::Other), Pipeline::default()
+            .set_tokenizer(Lindera { tokenizer }));
+
+        // TODO: define dict path for korean
+        pipeline_map.insert((Script::Hangul, Language::Other), Pipeline::default()
+            .set_tokenizer(Lindera { tokenizer }));
```

Comment:
You may [impl Default](https://doc.rust-lang.org/stable/std/default/trait.Default.html#how-can-i-implement-default) on your Lindera wrapper which create directly the lindera tokenizer internally.